app/src/main/java/com/boarbeard/generator/beimax/MissionImpl.java

```diff
@@ -179,7 +190,7 @@ public int getMissionPhaseLength(int phase) {
     public boolean generateMission() {
         // generate threats
         boolean generated = false;
-        int tries = 100; //maximum number of tries to generate mission
+        int tries = 500; //maximum number of tries to generate mission
```

Comment:
why did you need to up this?

Reply:
mistake left over from some of my testing. 

Also realised i missed out on docstrings and spotted some spelling mistakes in comments. I'll get that fixed up and update the PR